test/unit/app/helpers/admin/republishing_helper_test.rb

```diff
@@ -0,0 +1,16 @@
+require "test_helper"
+
+class Admin::RepublishingHelperTest < ActionView::TestCase
+  setup do
+    @first_row = bulk_content_type_index_rows.first
+  end
+
+  test "#bulk_content_type_index_rows capitalises the first letter of the bulk content type" do
+    assert_equal "All organisation 'About Us' pages", @first_row.first[:text]
```

Comment:
I think the assertion is the wrong away around here and below

Reply:
Can never get these the right way round, I tried putting the opposite of what I thought it should be 😅